wger/core/views/fitbit.py

```diff
@@ -0,0 +1,101 @@
+import requests
+import urllib
+import base64
+import os
+
+
+class FitBit:
+    """Class to handle all fitbit operation"""
+    # App settings from fitbit as regards the app
+    CLIENT_ID = os.getenv('CLIENT_ID')
+    CLIENT_SECRET = os.getenv('CLIENT_SECRET')
+    SCOPE = os.getenv('SCOPE')
+    REDIRECT_URI = os.getenv('REDIRECT_URI')
+    # Authorization and authentication URIs
+    AUTHORIZE_URI = os.getenv('AUTHORIZE_URI')
+    TOKEN_REQUEST_URI = os.getenv("TOKEN_REQUEST_URI")
+
+    def ComposeAuthorizationUri(self):
```

Comment:
Let's be consistent and use `camelCase` for the method names. Do this for the rest of the methods.